content/help/faq/distributed-tracing/istio-copy-headers.md

```diff
@@ -0,0 +1,9 @@
+---
+title: Why can't Istio propagate headers instead of the application?
+weight: 20
+---
+
+Although an Istio sidecar will process both inbound and outbound requests for an associated application instance, it has no implicit way of correlating
+the outbound requests to the inbound request that caused them. The only way this correlation can be achieved is if the application
+[propagates relevant information](/docs/tasks/telemetry/distributed-tracing/overview/#understanding-what-happened) (i.e. headers) from the inbound
```

Comment:
It'd be more accurate to refer to the section earlier that has "Header propagation may be accomplished through client libraries or manually".